pontoon/base/static/js/translate.js

```diff
@@ -2608,7 +2608,7 @@ var Pontoon = (function (my) {
     renderFailedChecks: function(failedChecks, messageOnly) {
       if (!messageOnly) {
         $('#save-anyway')
-          .toggleClass('approve', this.approvedNotSubmitted)
+          .toggleClass('approve', this.isApprovedNotSubmitted || false)
```

Comment:
Instead of using this confusing `|| false`, if your goal is to ensure you pass a boolean (which I don't understand, JS is a weakly typed language, you shouldn't need to bother doing that), you should instead use either `Boolean(myVar)` or `!!myVar`. I favor the former, which is in my opinion a lot clearer about the intention.

Reply:
Good call.

Here's why it needs to be a `Boolean` (not just truthy/falsy) value:
http://api.jquery.com/toggleclass/#toggleClass-className-state

Reply:
Oh, OK, that's surprising! In that case, I would really favor using `Boolean(myVar)`.